Haunted One.txt

```diff
@@ -0,0 +1,7 @@
+Name:Haunted One
+ManaCost:2 B
+Types:Legendary Enchantment Background
+S:Mode$ Continuous | Affected$ Creature.IsCommander+YouOwn | AddTrigger$ BecomesTapped | Description$ Commander creatures you own have “Whenever this creature becomes tapped, it and other creatures you control that share a creature type with it each get +2/+0 and gain undying until end of turn.”

```

Comment:
change `“`/`”` -> `"` (also in Oracle line)
pls add the reminder text to the Description$ too

forge doesn't like curly apostrophes or quotes - something to watch for if copy/pasting from Scryfall